app/javascript/pages/components/Calendar.jsx

```diff
@@ -158,17 +164,21 @@ class Calendar extends React.Component {
       }
 
       function ticker(svg) {
-        const now = svg.append('text')
+        const now = d3.select('#jobs-year')
```

Comment:
Would append still work if you just added the class or id attribute to the element?

Reply:
Refactored, it all now starts by appending to an empty div with class: jobs__subtitle